src/Resources/config/services/generators.xml

```diff
@@ -35,6 +37,16 @@
             <argument>@SyliusRefundPlugin/Download/creditMemo.html.twig</argument>
             <argument>%sylius.refund.template.logo_file%</argument>
         </service>
+
+        <service
+            id="Sylius\RefundPlugin\Factory\ShopBillingDataFactory"
+            decorates="sylius_refund.factory.shop_billing_data"
+            decoration-priority="256"
+            public="false"
+        >
+            <argument type="service" id="Sylius\RefundPlugin\Factory\ShopBillingDataFactory.inner" />
+        </service>
```

Comment:
Ok, that's why we do not specify a factory in `config.yml` 🤔 Maybe we should skip the decoration? We can do a factory + entry in resource configuration, the same way as it's done for `RefundPayment` (and it would keep an application consistency 💃 )

Reply:
Well I leaved comment in discussion above, but thing about specify a factory was something with this.

Reply:
I'd try to use a factory here, it shouldn't cause any errors in this relatively easy use-case.